### PR TITLE
fix error in showing plot

### PR DESCRIPTION
**Pull request by Sebaxp95**

### CODE REVIEW

**Comment by Sebaxp95** [comment]
[plotly component must remove all traces, not only the last one](https://trello.com/c/uQyAIna9/470-plotly-component-must-remove-all-traces-not-only-the-last-one)

File: src/app/plotly/plotly.component.ts [hunk 1]
```diff
@@ -58,21 +58,23 @@ export class PlotlyComponent implements OnInit, AfterViewInit, OnChanges {
       console.log(this.data);
       console.log('[PlotlyComponent] layout: ');
       console.log(this.layout);
-      const div = document.getElementById('plotly-' + this.randomId);
-      Plotly.deleteTraces(div, 0);
-      Plotly.addTraces(div, this.data);
+      this.createPlot();
```
Comment (gmrukwa) [inline src/app/plotly/plotly.component.ts]: Don't we need to delete the plot data somehow?

Comment (Sebaxp95) [reply src/app/plotly/plotly.component.ts]: No, newPlot remove and create new plot, but in newer version of plotly is function react which is more efficient than newPlot (https://plot.ly/javascript/plotlyjs-function-reference/#plotlyreact) so I added it.

Comment (Sebaxp95) [reply src/app/plotly/plotly.component.ts]: According to the documentation:
Plotly.react has the same signature as Plotly.newPlot above, and can be used in its place to **create a plot**, but when called again on the same <div> will update it far more efficiently than Plotly.newPlot

**Pull request merged**